src/frontend/hooks/server/mediaSync.ts

```diff
@@ -18,31 +13,31 @@ export function isArchiveDevice(value: MediaSyncSetting): boolean {
 }
 
 export function useGetMediaSyncSetting() {
-  const api = useClientApi();
+  const {data: isArchive} = useIsArchiveDevice();
+  const setting = convertMediaSyncSetting(isArchive);
 
-  return useSuspenseQuery({
-    queryKey: [MEDIA_SYNC_SETTING_KEY],
-    queryFn: async () => {
-      const isArchive = await api.getIsArchiveDevice();
-      return convertMediaSyncSetting(isArchive);
-    },
-  });
+  return {
+    data: setting,
+  };
```

Comment:
maybe more stylistic but can reduce this a bit if this is the only value that's needed

```suggestion
  return setting;
```

Reply:
ok done. Hope it's ok that I reduced it even more.